Makefile

```diff
@@ -51,13 +54,26 @@ test-coverage:
 	poetry run coverage report --show-missing
 	poetry run coverage xml
 
+generate-man:
+	@export QPC_VAR_CURRENT_YEAR=$(CURRENT_YEAR) \
+	&& jinja -X QPC_VAR docs/source/man.j2 $(ARGS)
+
+update-man.rst:
+	$(MAKE) generate-man ARGS="-o docs/source/man.rst"
+
+manpage-test:
+	@poetry run $(MAKE) generate-man | grep -v "Entering directory" | \
+	grep -v "Leaving directory" > output.txt
+	@diff -u docs/source/man.rst output.txt
```

Comment:
why manpage-test neesds these "grep -v"s and manpage role doesn't?

Reply:
good observation. It also needs to strip those undesired lines. I made the correction.